src/ZendCodingStandard/ruleset.xml

```diff
@@ -247,17 +292,13 @@
     <!-- Forbid use of longhand cast operators -->
     <rule ref="SlevomatCodingStandard.PHP.TypeCast"/>
     <!-- Require presence of declare(strict_types=1) -->
-    <rule ref="SlevomatCodingStandard.TypeHints.DeclareStrictTypes">
-        <properties>
-            <property name="newlinesCountBetweenOpenTagAndDeclare" value="2"/>
-            <property name="spacesCountAroundEqualsSign" value="0"/>
-            <property name="newlinesCountAfterDeclare" value="2"/>
-        </properties>
-    </rule>
+    <rule ref="WebimpressCodingStandard.Files.DeclareStrictTypes"/>
+    <!-- Format of strict type declaration -->
+    <rule ref="WebimpressCodingStandard.PHP.DeclareStrictTypes"/>
     <!-- Forbid useless parentheses -->
     <rule ref="WebimpressCodingStandard.Formatting.UnnecessaryParentheses"/>
-    <!-- Forbid useless semicolon `;` -->
-    <rule ref="SlevomatCodingStandard.PHP.UselessSemicolon"/>
+    <!-- Forbid redundant semicolons -->
+    <rule ref="WebimpressCodingStandard.PHP.RedundantSemicolon"/>
```

Comment:
Any reason for replacing these Slevomat tools with Webimpress ones, besides ownership?